File: 프로그래머스/0/181878. 원하는 문자열 찾기/원하는 문자열 찾기.py
def solution(myString, pat):
    a = "a"
    
    return 1 if pat.lower() in myString.lower() else 0

# 다른 풀이
return int(pat.lower() in myString.lower())

# 다른 풀이
answer = myString.lower().find(pat.lower())

File: 프로그래머스/0/181878. 원하는 문자열 찾기/원하는 문자열 찾기.java
class Solution {
    public int solution(String myString, String pat) {
        
        return myString.toLowerCase().contains(pat.toLowerCase()) ? 1 : 0;
    }
}

// 다른 풀이
int answer = 0;
String str = myString.toLowerCase();
String str2 = pat.toLowerCase();

if (str.indexOf(str2) != -1) {
    return 1;
}
return 0;
